src/Microsoft.TemplateEngine.Cli/Commands/NewCommand.Legacy.cs

```diff
@@ -35,6 +35,26 @@ internal partial class NewCommand : BaseCommand<NewCommandArgs>
             IsHidden = true
         };
 
+        internal IEnumerable<Option> LegacyOptions
+        {
+            get
+            {
+                return GetLegacyOptions();
```

Comment:
We don't need this local function. We can replace this call with the contents of the function.

Reply:
cool, I didn't know that yield works in property.